LibTrixi.jl/src/LibTrixi.jl

```diff
@@ -31,12 +31,24 @@ export trixi_nelements,
 export trixi_nelements_global,
        trixi_nelements_global_cfptr,
        trixi_nelements_global_jl
+export trixi_ndofs,
+       trixi_ndofs_cfptr,
+       trixi_ndofs_jl
+export trixi_ndofs_global,
+       trixi_ndofs_global_cfptr,
+       trixi_ndofs_global_jl
+export trixi_ndofs_element,
+       trixi_ndofs_element_cfptr,
+       trixi_ndofs_element_jl
 export trixi_nvariables,
        trixi_nvariables_cfptr,
        trixi_nvariables_jl
 export trixi_load_cell_averages,
        trixi_load_cell_averages_cfptr,
        trixi_load_cell_averages_jl
+export trixi_load_prim,
+       trixi_load_prim_cfptr,
+       trixi_load_prim_jl
```

Comment:
I am not a fan of arbitrary abbreviations unless there is a good reason for it. `cons2prim` and friends are named this way for historic reasons, and since they require a certain brevity. Here, on the other hand, I'd recommend to use something more descriptive, e.g., `trixi_load_primitive_variables` (or `_vars`, since that should be descriptive enough).